code/modules/mob/living/silicon/ai/malf.dm

```diff
@@ -57,14 +58,14 @@
 // Recalculates CPU time gain and storage capacities.
 /mob/living/silicon/ai/proc/recalc_cpu()
 	// AI Starts with these values.
-	var/cpu_gain = 0.01
+	var/cpu_gain = 0.1

```

Comment:
Probably buff shouldn't be so big. Needs to be tested.

Reply:
Yup, definitly needs testing, but its a easy fix if its too much so I´d leave it in and see how it develops.

Reply:
I just tried it out on a local testserver.
Around minute 30 I researched system override.
Which took another 24 minutes to complete.
At minute 39 the first hack announcement has been broadcasted.

It would allow powerplayers to nuke the station by the hour mark. (Which I am not really a fan of)

I can see two options to balance it:
- Increase the self-destruct time to 15 minutes
- Abort the self destruct, if the authentication disk is inserted into the self-destruct terminal and increase the time the self destruct takes.

Reply:
The self-destruct now takes 20 minutes and can be overwritten by inserting the auth disk into the nuke

Reply:
Now I can think of two scenarios if AI don't do anything about them:
 - Peaple decide to just abandon station, as 20 minutes is enough time to do so.
 - Just overrides nuke.
Anyway, this should be better than before. Still needs to be tested on larger scale, seeing how peaple factor changes things.

Reply:
Yes, I intentionally selected the time so a shuttle can be called / the nuke disarmed.
Once the AI has achieved station takeover it has a susbstantial advantage (as it can max out almost all of the other tech-trees while station-takeover is running)

But yes. It definitely needs testing.